### DISCUSSION TITLE
Campus experts 2023

### DISCUSSION THREAD

**Discussion by krishnapriya-n** [community/community#46334] [GitHub Education] [answered]
When do the Applications for Feb 2023 GitHub Campus Experts open? The attached image is what the page currently displays.

![image](https://user-images.githubusercontent.com/101842284/216737909-223ba7c4-5017-4733-a8fd-a7271df9062e.png)

**Comment by jge162** [comment]
Unfortunately you can just subscribe for updates. Not sure of an exact date anywhere.

**Reply by krishnapriya-n** [reply]
I see, thank you. I have subscribed but quite impatient haha.

**Reply by jge162** [reply]
understandable, good luck to you.

**Comment by abhuman** [comment]
Exactly, when it will open

**Comment by OgaDavid** [comment]
Registration is open now. you can register here: https://education.github.com/campus_experts

**Reply by Harish2303** [reply]
Showing me this "Your Student Developer Pack will be ready soon!" but I already had a Student Developer Pack. What do I have to do now? When will it be ready so I can fill out the form?

**Comment by melvincwng** [comment, marked as answer]
Hey @krishnapriya-n & @abhuman 👋, applications for the GitHub Campus Expert program are **now open till February 27, 2023 23:00 Pacific Time**.

You can apply for the GCE program via the new link over here: https://education.github.com/campus_experts.

The [old link](https://apply.githubcampus.expert/) that was used for application by previous batches is no longer in use and is defunct.

All the best in your GitHub Campus Experts application, and hope to see you as part of the GCE family very soon 😃

**Reply by BGMP** [reply]
Hi! I'm planning on applying too, but the "Apply Now!" button seems to not be working on my end... it just takes me back to https://education.github.com/campus_experts/status in an infinite loop. Any idea of why it might not be working for me? Thanks 😄

**Reply by Harish2303** [reply]
Showing me this "Your Student Developer Pack will be ready soon!" but I already had a Student Developer Pack. What do I have to do now? When will it be ready so I can fill out the form?

**Comment by abhuman** [comment]
hey @melvincwng  the Apply Now button is not working ...
![github campus application](https://user-images.githubusercontent.com/72157530/217170511-a36dff80-5512-4c7e-b2cb-d4e666f06b00.png)

**Comment by yakatyansh** [comment]
The apply button has started working

**Comment by abhuman** [comment]
Thank you Everyone @yakatyansh @krishnapriya-n @BGMP @jge162 @melvincwng

**Comment by AnuragParashar2000** [comment]
Hey, I was applying for GitHub campus expert from Guru Gobind Singh Indraprastha University Delhi, But I have heard that applications are getting rejected because on the GitHub Website it shows that the current expert is some one else, He is one of our senior who has passed out from the college long back on the year 2020. Please suggest me something on this.

**Comment by abhuman** [comment]
hey @AnuragParashar2000  
contact the previous campus expert , he/she will help you

**Reply by AnuragParashar2000** [reply]
Ok thank you

**Comment by Dinesh-Kumar-E** [comment]
when will be the results be announced?

**Reply by abhuman** [reply]
Today

**Reply by Dinesh-Kumar-E** [reply]
how is the selection process and how the selected participants will be informed? is it through mail?

**Comment by abhuman** [comment]
Yes

**Comment by yakatyansh** [comment]
did anyone Get a mail ??

**Reply by sher0u** [reply]
no Iam waiting

**Reply by levxn** [reply]
Yah same. The mail was supposed to be sent on the 15th of March. Yet to here from the campus expert team.

**Reply by sher0u** [reply]
we are all waiting i hope we will be selected this time .. 
this is my third time here trying (-.-)

**Reply by KingerNL** [reply]
Hi there, where did you guys found that an e-mail will be send on 15 of March? Or did you hear it from support? Can't find it myself

**Reply by KingerNL** [reply]
Ahhhh I see, thanks. Hopefully we'll get it soon.

**Reply by Pradeep-Kurapati** [reply]
I'm waiting too!

**Reply by cyberbuddy-manas** [reply]
Hey, did anyone get accepted?

**Reply by Pradeep-Kurapati** [reply]
No. Did you?

**Reply by cyberbuddy-manas** [reply]
> No. Did you?

Yeah, got rejected the third time

**Reply by KingerNL** [reply]
Dahm me too, what was your guys feedback?

**Comment by abhuman** [comment]
No

**Comment by DhruvLamba** [comment]
I still haven't recieved the result of GCE

**Comment by cyberbuddy-manas** [comment]
It's out, must be in the spam folder

**Reply by DhruvLamba** [reply]
I checked it everywhere but couldn't find it.

**Reply by cyberbuddy-manas** [reply]
> I checked it everywhere but couldn't find it.

Results are already out so you just have received it unless there's some technical glitch

**Reply by DhruvLamba** [reply]
Yeah there is some technical glitch

**Comment by Vikash-8090-Yadav** [comment]
Hello GitHub Education team,

I applied for the GitHub Campus Expert program during the February 2023 batch and was selected for the next round, which involved submitting a video resume. It has been two weeks since the submission closed, but I have not received any updates on my application status yet. I am very excited about this opportunity and couldn't resist reaching out to inquire about when I can expect to hear back.

Thank you for your time and consideration. I apologize for any errors in my previous message.

Best regards.
 @melvincwng

**Comment by harry-data-g** [comment]
Hello everyone!
When the results of last application of Github campus expert are going to be rolled out??

**Reply by BGMP** [reply]
they're slowly being rolled out I think, I received my response just a few minutes ago! 😄
